docs/pages/concepts/architectural/architecture.md

```diff
@@ -24,7 +24,8 @@ Meshery and its components are written using the following languages and technol
 | [Meshery Operator](/concepts/architecture/operator)                  | Golang                                                                            |
 | &nbsp;&nbsp;&nbsp;&nbsp; [MeshSync](/concepts/architecture/meshsync) | Golang                                                                            |
 | &nbsp;&nbsp;&nbsp;&nbsp; [Broker](/concepts/architecture/broker)     | Golang, NATS                                                                      |
-| [Meshery Database](/concepts/architecture/database)                  | Golang, SQLlite                                                                   |
+| [Meshery Database](/concepts/architecture/database)                  | Golang, SQLlite                                                                   | 
+| Meshery CLI                                                          | Golang                                                                            |
```

Comment:
Thanks. Please hyperlink (use an anchor) to the CLI section.

Reply:
Sure, Let me know if something else needs to be done.

Reply:
Yes, this will be the third time I've offered the feedback to please hyperlink (use an anchor) to the CLI section.